manifests/config.pp

```diff
@@ -86,6 +86,13 @@
     mode    => '0644',
   }
 
+  file { '/etc/pki/pulp/rsa.key':
+    ensure => file,
```

Comment:
If you remove this, then puppet won't create the file and thus not interfere with the ```%post```.